src/lycophron/logger.py

```diff
@@ -6,20 +6,31 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 """Lycophron logger class."""
 
-# TODO load  configs from a higher level file.
 # TODO logging is currently implemented without using a class-based approach.
 
-from logging import Logger, INFO, WARNING
+import logging
+import os
+import sys
 
-DEFAULT_LEVEL = INFO
 
+def init_logging(root_path=os.getcwd()):
```

Comment:
This is just moving code around and allowing the log file(s) to be created in a specific directory